### PR TITLE
Add note about why go mod replace is needed.

### PR DESCRIPTION
**Pull request by sipsma**
Signed-off-by: Erik Sipsma <erik@sipsma.dev>

### CODE REVIEW

File: docs/current/partials/_install-sdk-go.md [hunk 1]
```diff
@@ -9,6 +9,10 @@ go get dagger.io/dagger@latest
 go mod edit -replace github.com/docker/docker=github.com/docker/docker@v20.10.3-0.20220414164044-61404de7df1a+incompatible
 ```
 
+:::note
+The replace statement is currently needed due to one of Dagger's dependencies using a replace statement in their Go module. There is an issue tracking fixes to this [here](https://github.com/dagger/dagger/issues/3391).
```
Comment (vikram-dagger) [inline docs/current/partials/_install-sdk-go.md:13]:
```suggestion
The `replace` statement is currently needed due to one of Dagger's dependencies using a `replace` statement in its Go module. Learn more in this [GitHub tracking issue](https://github.com/dagger/dagger/issues/3391).
```

**Review by vikram-dagger — COMMENTED** [review]
Minor suggestion

**Review by vikram-dagger — APPROVED** [review]

**Comment by sipsma** [comment]
@vikram-dagger this doesn't seem to show up in the deploy preview: https://deploy-preview-3547--devel-docs-dagger-io.netlify.app/sdk/go/371491/install/

Am I missing something or messing up the syntax?

**Comment by sipsma** [comment]
@crjm and @jpadams confirmed locally that this renders, Netlify is just being weird. Thanks!

**Pull request merged**